www/js/app.js

```diff
@@ -1,22 +1,68 @@
 // Ionic E-Mission App
 
-// angular.module is a global place for creating, registering and retrieving Angular modules
-// 'emission' is the name of this angular module example (also set in a <body> attribute in index.html)
-// the 2nd parameter is an array of 'requires'
-// 'emission.services' is found in services.js
-// 'emission.controllers' is found in controllers.js
 'use strict';
 
-angular.module('emission', ['ionic',
+import angular from 'angular';
+import 'angular-animate';
+import 'angular-sanitize';
+import 'angular-translate';
+import '../manual_lib/angular-ui-router/angular-ui-router.js';
+import 'angular-local-storage';
+import 'angular-translate-loader-static-files';
+
+import 'moment';
+import 'moment-timezone';
+
+import i18next from 'i18next';
+
+import 'ionic-toast';
+import 'ionic-datepicker';
+import 'angular-simple-logger';
+
+import '../manual_lib/ionic/js/ionic.js';
+import '../manual_lib/ionic/js/ionic-angular.js';
+
+
+import en from '../i18n/en.json';
+import es from '../../locales/es/i18n/es.json';
+import fr from '../../locales/fr/i18n/fr.json';
+import it from '../../locales/it/i18n/it.json';
+const langs = { en, es, fr, it };
+
+let resources = {};
+for (const [lang, json] of Object.entries(langs)) {
+  resources[lang] = { translation: json }
+}
+
+const locales = !navigator?.length ? [navigator.language] : navigator.languages;
+let detectedLang;
+locales.forEach(locale => {
+  const lang = locale.trim().split(/-|_/)[0];
+  if (Object.keys(langs).includes(lang)) {
+    detectedLang = lang;
+  }
+});
+
+i18next.init({
+  debug: true,
+  resources,
+  lng: detectedLang,
+  fallbackLng: 'en'
+});
+
```

Comment:
No, it works without. I thought that too, but I think they're just exampling the common initialization of i18next.

Instead of `backend` we're supplying `resources`, which allows us to load offline translations (coming from our JSON files).

If we were going to explore locize or something https://github.com/e-mission/e-mission-phone/pull/974#discussion_r1206243608, *then* I think we'd need something else.